packages/clerk-js/src/ui/components/OrganizationProfile/MembersSearch.tsx

```diff
@@ -77,7 +78,14 @@ export const MembersSearch = ({ query, value, memberships, onSearchChange, onQue
 
   return (
     <Animated asChild>
-      <Flex sx={{ width: '100%' }}>
+      <Flex
+        sx={{
+          width: '50%',
+          [mqu.sm]: {
+            width: 'auto',
```

Comment:
On mobile we use `auto` to avoid the field being to small. 50% felt to small.

![Screenshot 2025-02-19 at 6 08 15 PM](https://github.com/user-attachments/assets/3e947106-9ece-4f44-97f4-8e86481ca102)